controller/array_action/array_mediator_ds8k.py

```diff
@@ -257,7 +259,7 @@ def _create_api_volume(self, name, size_in_bytes, space_efficiency, pool_id):
             )
             logger.info("Found volume {}".format(name))
             if api_volume is not None:
-                return api_volume
+                raise array_errors.VolumeAlreadyExists(name, self.identifier)
```

Comment:
I'm not sure it's good to change in "renaming" commits the program behavior. I suggest changing it in a different PR.

Reply:
this is a continuation of https://github.com/IBM/ibm-block-csi-driver/pull/301 (it was too large).
would it help if I change the PR title?

Reply:
Yes

Reply:
done